modules/machineset-gcp-shielded-vms.adoc

```diff
@@ -45,6 +45,11 @@ spec:
 <1> In this section, specify any Shielded VM options that you want.
 <2> Specify whether UEFI Secure Boot is enabled. Valid values are `Disabled` or `Enabled`.
 <3> Specify whether integrity monitoring is enabled. Valid values are `Disabled` or `Enabled`.
++
+[NOTE]
+====
+When integrity monitoring is enabled, you must not disable the virtual trusted platform module (vTPM).
+====
 <4> Specify whether virtual trusted platform module (vTPM) is enabled. Valid values are `Disabled` or `Enabled`.
```

Comment:
```suggestion
====
+
<4> Specify whether virtual trusted platform module (vTPM) is enabled. Valid values are `Disabled` or `Enabled`.
```
The 4th annotation is showing up as `<1>`. I think it's missing a `+` before it.

Reply:
Both great catches. Will wait for this fix to build and verify the output. TYSM!